lnwallet/channel.go

```diff
@@ -4116,19 +4107,21 @@ func (lc *LightningChannel) InitNextRevocation(revKey *btcec.PublicKey) error {
 
 // AddHTLC adds an HTLC to the state machine's local update log. This method
 // should be called when preparing to send an outgoing HTLC.
-func (lc *LightningChannel) AddHTLC(htlc *lnwire.UpdateAddHTLC) (uint64, error) {
+func (lc *LightningChannel) AddHTLC(htlc *lnwire.UpdateAddHTLC,
```

Comment:
The `godoc` comment should be updated to describe the semantics of the `openKey`.